README.txt

```diff
@@ -1,162 +1,89 @@
+INTRODUCTION
```

Comment:
Please use title casing, e.g. `Introduction`, all caps is usually not a good idea in documentation.